### PR TITLE
Remove advertising of deprecated functions and cleanup

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**

Fixes https://github.com/cyberbotics/webots/issues/4362

Since the functions `get_number_of_contact_points`, `get_contact_point` and `get_contact_point_node` have been deprecated in favor of `get_contact_points`, the former ones should not be advertised whereas the latter should.

This PR also cleans up a typo in the virtual reality service

### CODE REVIEW

**Comment by ad-daniel** [comment]
The issue with the md5 stemmed from the fact that when computing it we should ignore the arrays, however when doing the replacement with the custom message (which also was absent), it was done incorrectly.

The reason it surfaced only now is that in RosSupervisor, we mixed the names for the servers [here](https://github.com/cyberbotics/webots/blob/master/projects/default/controllers/ros/RosSupervisor.cpp#L81) (note the variable is called `mNodeGetContactPointServer` instead of `mNodeGetContactPoint>>s<<Server`). There used to be a `get_contact_point` service so it didn't raise any error.

File: projects/default/controllers/ros/headersGenerator.py [hunk 1]
```diff
@@ -72,13 +73,13 @@ def ros_md5sum(self, srv):
...
                 l = l.split('#')[0].strip()  # strip comments
-                if "geometry_msgs/" in l or "std_msgs/" in l or "sensor_msgs/" in l or "Header" in l:
-                    # replace message by its corresponding md5
+                if "geometry_msgs/" in l or "std_msgs/" in l or "sensor_msgs/" in l or "Header" in l or 'webots_ros/' in l:
                     message = l.split(' ')[0]
-                    # the fact that it is an array is not releavant for the computation of the MD5
-                    message = message.replace("[]", "")
-                    if message in self.predifinedMD5:
-                        l = l.replace(message, self.predifinedMD5[message])
+                    # replace message by its corresponding md5
```
Comment (omichel) [inline projects/default/controllers/ros/headersGenerator.py:78]: Consistency with the following comment:
```suggestion
                    # replace message by its corresponding MD5
```

**Review by omichel — COMMENTED** [review]
Thank you. I just have one minor comment.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**